lib/github_integration/install-branch.js

```diff
@@ -28,15 +29,21 @@ const takedown = async () => {
   }
 };
 
-
 const waitForPage = async (page) => {
   // Most reliable way we've found to be sure the server finishes restarting
   // This counts on the fact that docassemble makes a `test.yml` file
   // for every new project. This method doesn not care if the file errors.
   const tries = 20;
-  const test_url = `${puppeteerutils.BASE_INTERVIEW_URL}%3Atest.yml`;
-  console.log("interview url:", test_url);
-  console.log("This isn meant to make sure the interview will load. Sometimes it doesn't work. If this test errors because the interview wouldn't load, you can try re-running this test.");
+  
+  // So that we can try to load our interview, we need this file to exist and work
+  // DEVELOPER: Always make sure your test.yml doesn't create an error page
+  await page.goto(`${env_vars.BASE_URL}/playground?project=${env_vars.PROJECT_NAME}`, {waitUntil: 'domcontentloaded'});
+  return
```

Comment:
Why are we returning here? Doesn't that mean the subsequent lines will never get executed?

Reply:
I have no idea how this worked and now I'm a little worried...